wiki/List_of_Guides/Searching_and_Downloading_Beatmaps/zh.md

```diff
@@ -43,53 +45,65 @@ Osu!谱面搜索引擎对于关键词采用“或”的关系进行搜索，即
 
 所谓“尊重原作”，即输入歌曲最原本的相关信息，而不是使用译名/非官方名。
 
-`例1：`
-` `**`√`**` 初音ミク`
-` `**`×`**` 初音未来`
+```
+例1：
+ **√** 初音ミク
```

Comment:
Most likely a false alarm: https://osu.ppy.sh/wiki/ZH:Searching_and_Downloading_Beatmaps

Probably would want to delete all GFM styling in the code blocks.